src/main.py

```diff
@@ -14,7 +14,7 @@ async def main():
     Resources must be pre-registered. For an example, see the `__init__.py` file.
     """
     module = Module.from_args()
-    module.add_model_from_registry(Camera.SUBTYPE, OakDModel.MODEL)
+    module.add_model_from_registry(Camera.SUBTYPE, Oak.MODEL)
```

Comment:
where are the aliases?

I would expect
```suggestion
    module.add_model_from_registry(Camera.SUBTYPE, Oak.MODEL)
    module.add_model_from_registry(Camera.SUBTYPE, OakD.MODEL)
```

And in main 

```go
Registry.register_resource_creator(
    Camera.SUBTYPE,
    Oak.MODEL,
    ResourceCreatorRegistration(Oak.new, Oak.validate)
    
  Registry.register_resource_creator(
    Camera.SUBTYPE,
    OakD.MODEL,
    ResourceCreatorRegistration(Oak.new, Oak.validate)
```

So we're just wrapping the old code for all the models that are encompassed in this module.